runtimes/common/src/constants.rs

```diff
@@ -408,6 +408,18 @@ pub mod preimage {
 	}
 }
 
+pub mod tips {
+	use super::*;
+
+	parameter_types! {
+		pub const DataDepositPerByte: Balance = deposit(0, 1);
+		pub const MaximumReasonLength: u32 = 16384;
```

Comment:
This is naively taken from Kusama configuration, I could not find any information where this value comes from unfortunately.

Reply:
I think we were told once that there is a threshold up to which a storage entry size doesn't influence the weights. If this threshold is passed, the storage needs to be taken into account when doing benchmarks. Maybe the value is related to that?